libecho/src/Echo.cc

```diff
@@ -29,10 +29,15 @@ void Echo::initEcho(int a_argc, char **a_argv) {
 
 void Echo::send(const std::vector<uint8_t> &buffer) {
     auto encoded = converter->encode(buffer);
-    const size_t atOnce = 2600;  // (bitrate / notify_interval) + eps; - TODO: wyjaśnić
-
-    output->enqueueData(encoded.data(), std::min(encoded.size(), atOnce));
-    for (int i = atOnce; i < encoded.size(); i += atOnce) {
+    const size_t atOnce =
+        2600; /* < Temporary - this constant was created empirically. Shortly - it says how much audio data need to be
+                 in the pushed to stream buffer every time so there won't be pause in transmission.  */
+
```

Comment:
Should it really be 2600?
And maybe move it to previous line and break comment differently?

Reply:
I will leave the value like it is for now, we will look at it carefully later ^^